app/components/templates/menu-page/menu-page.component.tsx

```diff
@@ -0,0 +1,90 @@
+import {FC} from 'react';
+
+import {
+  CollectionSection,
+  Img,
+  MenuPageContainer,
+  Product as ProductFrame,
+  ProductsGrid,
+} from './menu-page.styles';
+import {Divider} from '../../atoms/divider';
+import {Hr} from '../../atoms/hr';
+import {Paragraph} from '../../atoms/paragraph';
+import {Subtitle} from '../../molecules/subtitle';
+
+export interface MenuProductPriceRange {
+  minVariantPrice: {amount: string; currencyCode: string};
+  maxVariantPrice: {amount: string; currencyCode: string};
```

Comment:
Why is the amount a string? is this what Shopify sends it as?

Reply:
I'm pretty sure it is but I'll check again just in case.

Reply:
Shopify handles the amount as a Decimal but in the editor it gives me an error whenever I define it as a number.

Looked it up and `amount` is serialized as a string: https://shopify.dev/docs/api/storefront/2023-01/scalars/Decimal